src/window.rs

```diff
@@ -199,17 +200,19 @@ impl Window {
     ///
     /// This creates a window and fully initializes a window.
     pub fn new(
-        title: &str,
+        options: &Options,
         window_config: &WindowConfig,
     ) -> Result<Window> {
         let event_loop = EventsLoop::new();
 
+        let title = &options.title;
+
         let window_builder = WindowBuilder::new()
-            .with_title(title)
+            .with_title(title.to_owned())
```

Comment:
You can actually use `&*options.title` here instead of defining a temporary variable.